models/unet-2d-nuclei-broad/UNet2DNucleiBroad.model.yaml

```diff
@@ -41,39 +41,37 @@ outputs:
 
 prediction:
     preprocess:
-        - spec: https://github.com/bioimage-io/pytorchbioimageio:configurations/transformations/NormalizeZeroMeanUnitVariance.transformation.yaml:9fec4dd
-          kwargs:
-              eps: 1.0e-6  # currently all kwargs need to be specified (here or in the consumer)
-              apply_to: [0]
+        - spec: https://raw.githubusercontent.com/FynnBe/pytorch-bioimage-io/pybio/specs/transformations/NormalizeZeroMeanUnitVariance.transformation.yaml
```

Comment:
I find this a bit ugly. Referenceing the file in the 'normal' github address instead of the raw file would be much nicer, and also easier for checking out specific commits, I think.
For reference, here's my hacky first implementation for  this:
https://github.com/bioimage-io/example-unet-configurations/blob/master/models/unet-2d-nuclei-broad/dummy_config_parser.py#L22

Reply:
agreed, this was simply the most general form, so the source could live anywhere, but we need the whole repo anyway for relative imports.
We could even allow for the #L<line_number> syntax (and then read and parse the content of that line to get the object name (additionally #<object_name> should be an option anyway). maybe the line resolution should only be allowed for versions specified by hash

Reply:
>  so the source could live anywhere

I personally wouldn't allow for that and would require it to live somewhere that can host git repos. I think that's a totally reasonable requirement.
Then we also don't need to implement the more general cases.

Reply:
looking at your posted file link and the overall awesomeness of github I suggest that we keep the source format as a valid url (and not add the git commit id at the very end):
```yaml
source: https://github.com/<orga_name>/<repo_name>/blob/<git commit id>/<path_to_file>#<fragment>
```
the fragment is only processed by the browser. I like the L<line_nubmer> notation, which we could use to infer the object_name (`def ... :` or `class ...:` in that line) additionally it would make a lot of sense to simply type #<object_name>, especially for branch names etc. where line numbers might change